ait/gui/static/js/ait/gui/Field.js

```diff
@@ -285,7 +291,8 @@ const Field =
     },
 
     onbeforeupdate (vnode, old) {
-        return this.hasChanged()
+        let changed = this.hasChanged()
```

Comment:
Any reason for this change?

Reply:
I think something I was messing with during dev but can be switched back now.